src/scss/components/_mobile-pop-over.scss

```diff
@@ -0,0 +1,78 @@
+/**
+ * Mobile pop overs
+ * --------------------------------
+ * Used for mobile filters pop over
+ *
+ * ================================= */
+
+$mobilePopOver-background       : $white;
+$mobilePopOver-padding          : spacing(x2);
+$mobilePopOver-border-color     : $grey--lightest;
+$mobilePopOver-shadow-color     : rgba($black, 0.12);
+
+.c-mobilePopOver {
```

Comment:
Best not to name things with device types in mind.  There might come a time where this becomes needed on other devices and then the name would become defunct.

Maybe naming it as something like a fullScreen popover and then keeping the comment description where it's currently used as an example would be good